HUBER/iocBoot/iocHUBER-IOC-01/config.xml

```diff
@@ -4,8 +4,12 @@
 <ioc_desc>Huber Controller</ioc_desc>
 <ioc_details></ioc_details>
 <macros>
+<macro name="PORT" pattern="^COM[0-9]+$" description="Serial COM port" hasDefault="NO" />
+<macro name="EMULATOR_PORT" pattern="^[0-9]+$" description="IP address of Huber controller" defaultValue="57677" hasDefault="YES" />
 <macro name="MTRCTRL" pattern="^[0-9]{1,2}$" description="Controller number used in motor address assignment" hasDefault="NO" />
 <macro name="NUMBERAXES" pattern="^([1-9]|(1[1-6]))$" description="Number of Axes on the Controller" defaultValue="5" hasDefault="YES" />
+<macro name="BAUD" pattern="^[0-9]+$" description="Baud rate (default: 19200)" defaultValue="9600" hasDefault="YES" />
+<macro name="PARITY" pattern="^(even|odd|none)$" description="Parity (default: none)" defaultValue="none" hasDefault="YES" />
```

Comment:
I guess you want to add STOP here if exposing the other COM settings

Reply:
that's if they are device-configurable! if not I think i'd just hide them and put sensible defaults in st.cmd

Reply:
They are device configurable

Reply:
ok, in that case i'd expose STOP as well